packages/airnode-deployer/src/handlers/gcp/index.ts

```diff
@@ -31,6 +40,9 @@ export async function startCoordinator(_req: Request, res: Response) {
   });
   await handlers.startCoordinator(parsedConfig, coordinatorId);
   const response = { ok: true, data: { message: 'Coordinator completed' } };
+  // TODO: line below (suspiciously?) not present in original PR:
+  // https://github.com/api3dao/airnode-qrng/commit/75c54a0f5dddbd2a29b7f303310356750d2fd1fc#diff-307ba728f020f0438491dbae15a76ec9421457225fcf4f4966f698b74a87263cR4-R8
+  caching.syncFsSync();
```

Comment:
I wonder whether we need this and whether OS doesn't provide some guarantees for this. But it might just be forgotten in the original PR.

Reply:
I think I found the reason: on GCP `/tmp` [is in RAM](https://cloud.google.com/appengine/docs/standard/go/using-temp-files)